### PR TITLE
[full-ci] Fix response code when upload a file over locked

### PR DESCRIPTION
**Pull request by 2403905**
## Description

We fixed a bug where the response code was incorrect when uploading a file over a locked file.

## Related Issue

- Fixes [#7638](https://github.com/owncloud/ocis/issues/7638)

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiLocks/lockFiles.feature [hunk 1]
```diff
@@ -312,7 +312,7 @@ Feature: lock files
       | new              |
       | spaces           |
 
-  @issue-7638
+    @issue-7638
```
Comment (phil-davis) [inline tests/acceptance/features/apiLocks/lockFiles.feature:315]:
```suggestion
  @issue-7638
```

The indentation change causes the linter to fail this feature file, and so the tests do not even run.

File: tests/acceptance/features/apiLocks/unlockFiles.feature [hunk 2]
```diff
@@ -63,7 +63,6 @@ Feature: unlock locked items
       | new              |
       | spaces           |
 
-  @issue-7638
```
Comment (saw-jan) [inline tests/acceptance/features/apiLocks/unlockFiles.feature:66]: @2403905 Let's not remove the issue tags. it servers as a reference for the future.

File: tests/acceptance/features/apiLocks/lockFiles.feature [hunk 3]
```diff
@@ -312,7 +312,6 @@ Feature: lock files
       | new              |
       | spaces           |
 
-  @issue-7638
```
Comment (saw-jan) [inline tests/acceptance/features/apiLocks/lockFiles.feature:315]: @2403905 Let's not remove the issue tags. it servers as a reference for the future.

File: tests/acceptance/features/apiLocks/lockFiles.feature [hunk 4]
```diff
@@ -370,7 +369,7 @@ Feature: lock files
       | spaces           | shared     |
       | spaces           | exclusive  |
 
-  @issue-7638 @issue-7599
```
Comment (saw-jan) [inline tests/acceptance/features/apiLocks/lockFiles.feature:373]: @2403905 Let's not remove the issue tags. it servers as a reference for the future.

File: tests/acceptance/features/apiLocks/lockFiles.feature [hunk 5]
```diff
@@ -334,7 +333,7 @@ Feature: lock files
...
```
Comment (saw-jan) [inline tests/acceptance/features/apiLocks/lockFiles.feature:337]: @2403905 Let's not remove the issue tags. it servers as a reference for the future.

File: tests/acceptance/features/apiLocks/lockFiles.feature (hunk 3 above)
Comment (phil-davis) [reply tests/acceptance/features/apiLocks/lockFiles.feature:315]: And if the tags do get removed, then leave a blank line (2 lines between scenarios)
If the line numbers of scenarios change, then it is a nuisance for the expected-failures files.

**Review by micbar — APPROVED** [review]

**Pull request merged**